packages/projection/src/operators/Mappers/certificates/withStakeKeyRegistrations.ts

```diff
@@ -19,7 +19,10 @@ export const withStakeKeyRegistrations = unifiedProjectorOperator<WithCertificat
     ...evt,
     stakeKeyRegistrations: evt.certificates
       .map(({ pointer, certificate }): StakeKeyRegistration | null => {
-        if (certificate.__typename === Cardano.CertificateType.StakeRegistration) {
+        if (
+          certificate.__typename === Cardano.CertificateType.StakeRegistration ||
+          certificate.__typename === Cardano.CertificateType.Registration
```

Comment:
arent we missing here also:

- StakeRegistrationDelegation
- StakeVoteRegistrationDelegation

These certificates also register the stake key (and do some additional work)

Reply:
Nice catch, @AngelCastilloB 💪 . I will update it.